policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax_credit/mi_homestead_non_refundable.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class mi_homestead_non_refundable(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan non-refundable homestead property tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax_credit
+
+        total_household_resources = tax_unit("mi_household_resources", period)
+
+        # disabled
+        disabled_people = add(tax_unit, period, ["is_disabled"])
```

Comment:
I am pretty sure we want to use `disabled_head` but please confirm with legal code or tax form